core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -134,4 +134,10 @@ private TableProperties() {
 
   public static final String ENGINE_HIVE_ENABLED = "engine.hive.enabled";
   public static final boolean ENGINE_HIVE_ENABLED_DEFAULT = false;
+
+  public static final String ROW_LEVEL_OPS_ISOLATION_LEVEL = "row-level-ops.isolation-level";
+  public static final String ROW_LEVEL_OPS_ISOLATION_LEVEL_DEFAULT = "serializable";
+
+  public static final String ROW_LEVEL_OPS_MODE = "row-level-ops.mode";
+  public static final String ROW_LEVEL_OPS_MODE_DEFAULT = "copy-on-write";
```

Comment:
Similar, how about `write.spark.mode`?

Reply:
I'll be happy to reconsider the naming but `write.spark.mode` seems to generic to me. If I read it in the docs, it would not be immediately clear to me that it applies only to UPDATE/DELETE/MERGE.

Reply:
Any prefix other than `row-level-ops` that can make it more descriptive?

Reply:
I like to use `write` because it will control behavior at write time. If we want this to be a general setting to prefer copy-on-write, then we could use `write.row-level.mode`. I think that's specific. What do you think?